ac_dc/oscar_sample_filter.py

```diff
@@ -105,13 +141,33 @@ def remove_long_words(
         filtered_sentence = " ".join(words)
         return filtered_sentence
 
+    @staticmethod
+    def apply_regex_govt_id_anonymization(sentence, lang_id):
+        for regex in list(
+            ModifyingSentences.govt_id_regex.get(
+                lang_id, ModifyingSentences.govt_id_regex["any"]
+            ).values()
+        ):
+            matched = False
+            for ent in regex.findall(sentence):
+                if type(ent) != str:
+                    continue
+                ent2 = ent.translate(ModifyingSentences.trannum)
+                text = text.replace(ent, ent2)
```

Comment:
Didn't we say we need to mark the data before deciding any replacement that could damage the original content?

Reply:
I think for the very risky stuff we would want to anonymize and this is quicker to meet our deadline in (2-5 days). But I'm fine with marking this first if we the data is kept internal and then anonymize. Please feel free to add code for this if you are able.